Sources/JavaScriptKit/FundamentalObjects/JSFunction.swift

```diff
@@ -84,18 +83,18 @@ public class JSFunction: JSObject {
     }
 }
 
-private func invokeNonThrowingJSFunction(_ jsFunc: JSFunction, arguments: [ConvertibleToJSValue], this: JSObject?) -> JSValue {
+internal func invokeNonThrowingJSFunction(_ jsFunc: JSFunction, arguments: [ConvertibleToJSValue], this: JSObject?) -> RawJSValue {
```

Comment:
In my understanding, `internal` is usually omitted as it's implicit. Is its presence warranted here?